### PR TITLE
Camp Specific Patrols

### PR DESCRIPTION
**Pull request by DeadGirlRisen**
Allows a person to specify the patrol can only happen in the specified camp.

Made on request of @Tiritirimatangi ^^

### CODE REVIEW

File: resources/game_config.json [hunk 1]
```diff
@@ -250,7 +250,7 @@
 		"best_stat_modifier": 10,
 		"fail_stat_cat_modifier": -15,
 		"chance_of_romance_patrol": 16,
-		"debug_ensure_patrol_id": null,
+		"debug_ensure_patrol_id": "bch_bord_camptest",
```
Comment (Tiritirimatangi) [inline resources/game_config.json]: You've left the testing patrol active here!

**Review by Tiritirimatangi — CHANGES_REQUESTED** [review]
Just a lil change - will also request another review from a more coding based senior dev <3

File: resources/game_config.json (hunk 1 above)
Comment (DeadGirlRisen) [reply resources/game_config.json]: Ooop! my bad XD will fix

**Review by Lixxis — APPROVED** [review]

**Review by Thlumyn — APPROVED** [review]

**Comment by Thlumyn** [comment]
@Tiritirimatangi merge whenever you are ready

**Review by Tiritirimatangi — APPROVED** [review]

**Pull request merged**